playbooks/lets-encrypt.yml

```diff
@@ -0,0 +1,58 @@
+---
+# lets-encrypt.yml asks questions about Streisand domain name, which is used to
```

Comment:
typo: "about Streisand domain name" -> "about the Streisand instance domain name" maybe?